### ISSUE TITLE
Stylize out REST API's docs

### ISSUE THREAD

**Issue by carlspring** [strongbox/strongbox#1126]
# Task Description

As a follow-up to #1124 and #1125, we would like to change the style of the docs to match that of our project.

Please note, that this task depends on #1124 and #1125 and that it cannot be implemented without them being done first.

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @fuss86

**Comment by matias9477** [comment]
Hello, I'd like to help in this taks, how can I do it?

**Comment by steve-todorov** [comment]
Hi @matias9477! We appreciate your interest in this task! However it is currently `on hold` and I will be picking it up once all of the required other tasks have been completed. Please have a look at the other available issues and feel free to join [our chat](https://chat.carlspring.org/channel/community) if you have questions or need help! :)

**Comment by dawsonnnnnnnn** [comment]
Hi! It looks like there's some folks being assigned and unassigned to this. I'd love to help out with the REST API docs once and for all  😁 Let me know if I could jump in.

**Comment by ggold7046** [comment]
Hi, could any body guide me to the rest api document file link?
